docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='add_project_{context}']
+= Creating a project
+
+You create a project using the *Create project* wizard.
+
+.Procedure
+
+. In the {WebName}, click *Projects*.
+. Click *Create project*.
++
+.Project details screen
+image::web-add-project_51.png[Project details screen]
+
+. Enter a unique name for the project, an optional description, and click *Next*.
++
+.Applications screen
+image::web-add-apps_51.png[Applications screen]
++
+. To upload applications, click *Browse*, select the application files you want to upload, and click *Close*.
++
+Uploading applications stores them directly in the {ProductShortName} server.
+
+. To register a server path, click the *Server path* tab and enter the *Server-side path* of the application in the field.
++
+Registering the server path of the file tells the {ProductShortName} server the location of the application. This method is useful if you expect to update the application because it ensures that the {ProductShortName} server always uses the latest version of the application.
+
+ . Click *Next*.
++
+.Transformation target screen
+image::web-configure-analysis_51.png[Transformation target screen]
+
+. Click the desired transformation targets.
+. Click *Next*.
++
+You can specify which Java application packages to include or exclude on the *Select applications* screen.
```

Comment:
This 'intro stuff' would belong to the next step but I think it's unnecessary. I would rewrite the actual step. So you can remove the "You can specify..." because it's obvious.